src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/PackageFeeds/MultiSourcePackageFeed.cs

```diff
@@ -178,6 +178,8 @@ public async Task<SearchResult<IPackageSearchMetadata>> RefreshSearchAsync(Refre
             TelemetryState telemetryState,
             CancellationToken cancellationToken)
         {
+            cancellationToken.ThrowIfCancellationRequested();
```

Comment:
This looks like a product change unrelated to telemetry.

What are you fixing?  What is expected and actual behavior (in PM UI)?

Reply:
It's a good pattern to respect any cancellation requests here. 
Not fixing anything in particular.